libdash/libdash/README_mpd.txt

```diff
@@ -0,0 +1,12 @@
+DASH-IF/Github repository - New Elements Test:
```

Comment:
What value does this new file provide? Seems there is a status "working" noted on all the added manifests. What does that mean? How was that tested?

If you think this files is not that important then just delete it. If you think it is important, I think it would be worth to add some more description to it and transform it to the MD format. Probably also good to link it from the root level README.md file?

Reply:
This file had been created with the intent of keeping trace of the evaluation and test of the new added elements - together with the tested MPD file - but it is not so relevant anymore to the update itself by the time every new added element is working as expected. I have already provided to the file removal.